student/static/student/css/registration.css

```diff
@@ -43,8 +43,16 @@ ul {
     margin: 10px 0px 10px 0px;
 }
 
+.basics-registration-form #original-price{
```

Comment:
@samklq Do we have Prettier or equivalent linting?

Reply:
Might be good to add as Git hook so when others start to contribute we'll have consistent formatting